test/tests/neutronics/tally_system/tests

```diff
@@ -0,0 +1,49 @@
+[Tests]
+  [multi_global_estimator]
+    type = Exodiff
+    input = multi_estimator.i
+    exodiff = multi_estimator_out.e
+    requirement = "The system shall correctly normalize local tallies with different estimators using multiple global tallies."
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [multiple_tallies]
+    type = Exodiff
+    input = multi.i
+    exodiff = multi_out.e
+    requirement = "The system shall correctly apply and normalize two different local tallies with different scores. "
```

Comment:
Nice! This is a great test for cell tallies. It may be useful to do the same type of test, but for (i) mesh tallies, (ii) relaxation, and (iii) triggers just to be extra sure we're not missing anything.

Reply:
Sounds good, I'll add some extra tests to cover those cases. When it comes to tally relaxation, should I add a case for each relaxation scheme or would constant relaxation work just fine?